src/style.css

```diff
@@ -1 +1,70 @@
-/* Styles go here */
+body {
+  font-family: "Roboto", Arial, Helvetica, sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+  margin: 0;
+}
+
+[class*="__link"] {
```

Comment:
Never do so, just enter the full class name
later nobody will be able to find it in CSS file if you have a selector like this